### ISSUE TITLE
Populate Restaurant Info Modal

### ISSUE THREAD

**Issue by emswann** [emswann/still-open#14]
Code Ajax call to Yelp API.

Create required variables.
Create query.
Call Yelp via Ajax.
Assign values to variables.

Dependent on obtaining API key.

Need to work around CORS issue. Use for displaying restaurant info and picture in modal.

**Comment by robertbernardbrown** [comment]
Found workaround using CORS Anywhere Heroku App. Will work on getting fields we need from Yelp API

**Comment by emswann** [comment]
@robertbernardbrown 

Is this a duplicate of Issue #32?

**Comment by emswann** [comment]
Changed issue to track populating restaurant info fields. Yelp API call is handled by Issue #39.

**Comment by emswann** [comment]
Yep. Still a duplicate of #32 and #38. Added duplicate label and closing.
